### ISSUE TITLE
NullPointerException in JSONTokener

### ISSUE THREAD

**Issue by JoostvDoorn** [JoostvDoorn/GlutenVrijApp#1]
```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:299)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:352)
at java.util.concurrent.FutureTask.setException(FutureTask.java:219)
at java.util.concurrent.FutureTask.run(FutureTask.java:239)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1080)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:573)
at java.lang.Thread.run(Thread.java:856)
Caused by: java.lang.NullPointerException
at org.json.JSONTokener.nextCleanInternal(JSONTokener.java:116)
at org.json.JSONTokener.nextValue(JSONTokener.java:94)
at org.json.JSONObject.<init>(JSONObject.java:154)
at org.json.JSONObject.<init>(JSONObject.java:171)
at com.joostvdoorn.glutenvrij.Search.search(Search.java:51)
at com.joostvdoorn.glutenvrij.Search.doInBackground(Search.java:83)
at com.joostvdoorn.glutenvrij.Search.doInBackground(Search.java:22)
at android.os.AsyncTask$2.call(AsyncTask.java:287)
at java.util.concurrent.FutureTask.run(FutureTask.java:234)
... 3 more
```

**Comment by JoostvDoorn** [comment]
Possible fix in 3f7dcff

**Comment by JoostvDoorn** [comment]
Possibly fixed in ac2febe4702f5df87d14cc67d6a059cab3acd499
